backend/game/player/player.py

```diff
@@ -32,3 +37,15 @@ def update_energy_by(self, change_integer):
         self.energy += change_integer
         if self.energy < constants.DEFAULT_ENERGY_CUBE:
             self.energy = constants.DEFAULT_ENERGY_CUBE
+
+    def add_card(self, card):
+        self.card_on_hand.append(card)
+
+    def remove_card(self, index):
+        self.card_on_hand.pop(index)
+
+    def get_card_at_index(self, index):
```

Comment:
I would prefer a different access style then doing everything by numeric index, think about how this is being utilized.  You're not going to be saying get_card_at_index(0), the player is most likely to say "get_keep_card(KEEP_CARD_NAME)"